tools/PowerShell/Microsoft.WinGet.Client/src/Library/Get-LanguageData.ps1

```diff
@@ -0,0 +1,34 @@
+Function Get-LanguageData {
+    PARAM(
+        [Parameter(Position=0)] $Language = (Get-UICulture).Name
+    )
+
+    $languageData = $(
+        $hash = @{}
+
+        $(try {
+            # We have to trim the leading BOM for .NET's XML parser to correctly read Microsoft's own files - go figure
+            ([xml](((Invoke-WebRequest -Uri "https://raw.githubusercontent.com/microsoft/winget-cli/master/Localization/Resources/$Language/winget.resw" -ErrorAction Stop ).Content -replace "\uFEFF", ""))).root.data
```

Comment:
Will this work if somebody has the same language but different culture? Say, if I use es-MX for Windows, I would probably want it to use the es-ES resources as there is no es-MX for winget

Reply:
I updated the module with the idea exposed. I let you check if it's clean or if there is a better way :)